src/plotting.jl

```diff
@@ -0,0 +1,430 @@
+    return series!(subfig, per_class_prg_curves, auc_labels; legend=legend, title=title,
+                           xlabel=xlabel, ylabel=ylabel)
+end
+
+function plot_roc_curves!(subfig::FigurePosition, per_class_roc_curves::SeriesCurves,
+                          per_class_roc_aucs::NumberVector,
+                          class_labels::AbstractVector{<: String};
+                          legend=:rb,
+                          title="ROC curves",
+                          xlabel="False positive rate",
+                          ylabel="True positive rate")
+
+    auc_labels = [@sprintf("%s (AUC: %.3f)", class, per_class_roc_aucs[i])
+                  for (i, class) in enumerate(class_labels)]
+
+    return series!(subfig, per_class_roc_curves, auc_labels; legend=legend, title=title,
+                           xlabel=xlabel, ylabel=ylabel)
+end
+
+function plot_reliability_calibration_curves!(subfig::FigurePosition,
+                                              per_class_reliability_calibration_curves::SeriesCurves,
+                                              per_class_reliability_calibration_scores::NumberVector,
+                                              class_labels::AbstractVector{String};
+                                              legend=:rb)
+
+    calibration_score_labels = map(enumerate(class_labels)) do (i, class)
+        @sprintf("%s (MSE: %.3f)", class, per_class_reliability_calibration_scores[i])
+    end
+
+    ax = series!(subfig, per_class_reliability_calibration_curves, calibration_score_labels;
+                         legend=legend, title="Prediction reliability calibration",
+                         xlabel="Predicted probability bin", ylabel="Fraction of positives",
+                         scatter=(markershape=Circle, markersize=5, strokewidth=0))
+    #TODO: mean predicted value histogram underneath?? Maybe important...
+    # https://scikit-learn.org/stable/modules/calibration.html
+    linesegments!(ax, [0, 1], [0, 1]; color=(:black, 0.5), linewidth=2, linestyle=:dash, label="Ideal")
+    return ax
+end
+
+function plot_binary_discrimination_calibration_curves!(subfig::FigurePosition, calibration_curve::SeriesCurves, calibration_score,
+                                                        per_expert_calibration_curves::SeriesCurves,
+                                                        per_expert_calibration_scores, optimal_threshold,
+                                                        discrimination_class::AbstractString;
+                                                        markershape=Rect, markersize=5)
+    ax = series!(subfig, per_expert_calibration_curves, nothing; legend=nothing,
+                         title="Detection calibration", xlabel="Expert agreement rate",
+                         ylabel="Predicted positive probability", color=:darkgrey,
+                         scatter=(markershape=markershape, markersize=markersize))
```

Comment:
```suggestion
                         scatter=(markershape=markershape, markersize=markersize, strokewidth=0))
```
...I think this is the change to remove the circle outlines on the discrimination calibration plot, so that it matches the reliability calibration plot?

Reply:
Good catch, thank you! :)